crates/astria-bridge-withdrawer/tests/blackbox/helpers/test_bridge_withdrawer.rs

```diff
@@ -75,20 +75,40 @@ pub(crate) const SEQUENCER_CHAIN_ID: &str = "test-sequencer";
 const ASTRIA_ADDRESS_PREFIX: &str = "astria";
 
 static TELEMETRY: LazyLock<()> = LazyLock::new(|| {
+    let config = Config {
```

Comment:
Can you move this and the config padded to the "test bridge withdrawer" to a free standing `fn config() -> Config` so they share the same definition?

Reply:
They do not share the same definition, the other config relies on items instantiated during the test and can't be a static item.